Tests/com/SIBV/FahrenheitToCelsiusTest.java

```diff
@@ -0,0 +1,17 @@
+package com.SIBV;
+
+import com.SIBV.FahrenheitToCelsius;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FahrenheitToCelsiusTest {
+    FahrenheitToCelsius v = new FahrenheitToCelsius();
+
+    @Test
+    void change() {
```

Comment:
it's usually a good idea to have a test method for each run of the code that you use to test.

So this would be better split into "temperaturePositive" and "temperatureNegative".

If you wanted to reuse the same code you could have a look at tests with parameters. It's a bit more complicated but interesting